<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<title>Laboratory of Computational Molecular Biology, BNU</title>
<meta http-equiv="Content-Type" content="text/html; charset=gb2312">
<style type="text/css">
<!--
.titleLink{
	font-family: "Arial", "Helvetica", "sans-serif";
	font-size: 14px;
	color: #FFFFFF;
	text-decoration: none;
	font-weight: bold;
}
.titleLink A:link {text-decoration: none;color: #FFFFFF;}
.titleLink A:visited {text-decoration: none;color: #FFFFFF;}
.titleLink A:active {text-decoration: none;color: #FFFFFF;}
.titleLink A:hover {text-decoration: underline; color: FFFFFF;}
.menu{
	font-family: "Arial", "Helvetica", "sans-serif";
	font-size: 13px;
	color: #000066;
	text-decoration: none;
	font-weight: bold;
}
.menu A:link {text-decoration: none;}
.menu A:visited {text-decoration: none;}
.menu A:active {text-decoration: none;}
.menu A:hover {text-decoration: underline;}
.mainLink{
	font-family: "Arial", "Helvetica", "sans-serif";
	font-size: 13px;
	color: #000066;
	text-decoration: none;
}
.mainLink A:link {text-decoration: none;}
.mainLink A:visited {text-decoration: none;}
.mainLink A:active {text-decoration: none;}
.mainLink A:hover {text-decoration: underline;}
.STYLE1 {font-family: Arial, Helvetica, sans-serif;
	font-size: 15px;
}
.STYLE12 {font-size: 17px; font-family: Arial, Helvetica, sans-serif;}
.STYLE5 {	font-family: Arial, Helvetica, sans-serif;
	font-size: 14px;
}
.STYLE9 {font-family: Arial, Helvetica, sans-serif; font-weight: bold; font-size: 17px; }
.STYLE13 {
	font-family: Arial, Helvetica, sans-serif;
	font-size: 25px;
	font-weight: bold;
}
.style14 {font-size: 15px}
.style15 {font-family: Arial, Helvetica, sans-serif}
.style17 {font-family: Arial, Helvetica, sans-serif; font-size: 16px; }
-->
</style>
</head>

<body BGCOLOR=#FFFFFF LEFTMARGIN=1 rightmargin=1 TOPMARGIN=1 MARGINWIDTH=1 MARGINHEIGHT=0>
<table width="100%" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td height="16" bgcolor="#000099"><table width="100%" border="0" cellpadding="0" cellspacing="0" background="images/title_bg_layer.jpg">
        <tr>
          <td width="6%" height="10"></td>
          <td width="15%"></td>
          <td width="79%"></td>
        </tr>
        <tr>
          <td height="28">&nbsp;</td>
          <td><img src="../images/bnu_log.jpg"></td>
          <td><h2><font color="#FFFFFF" face="Arial, Helvetica, sans-serif">Laboratory 
              of Computational Molecular Biology</font></h2>
            <font color="#FFFFFF" size="+1" face="Arial, Helvetica, sans-serif"><strong>College 
            of Life Sciences, Beijing Normal University</strong></font></td>
        </tr>
        <tr>
          <td height="10"></td>
          <td></td>
          <td></td>
        </tr>
      </table></td>
  </tr>
  <tr> 
    <td valign="top"><table width="100%" height="100%" border="1" cellpadding="1" cellspacing="1" bordercolor="#FFFFFF">
        <tr> 
          <td height="22" bgcolor="#0066FF"><table width="95%" border="0" cellpadding="0" cellspacing="0" bordercolor="#FFFFFF">
              <tr> 
                <td width="43" height="22">&nbsp;</td>
                <td width="91"><div align="center"> <a href="../index.htm" class="titleLink">Home</a></div></td>
                <td width="22"><div align="center"><font size="2"></font></div></td>
                <td width="67"><div align="center"> <a href="../faculty.htm" class="titleLink">Faculty 
                    </a></div></td>
                <td width="26">&nbsp;</td>
                <td width="96"><div align="center"><a href="../students.htm" class="titleLink">Students</a></div></td>
                <td width="13"><div align="center"><font size="2"></font></div></td>
                <td width="145"><div align="center"><a href="../research.htm" class="titleLink">Research 
                    Areas</a></div></td>
                <td width="21"><div align="center"></div></td>
                <td width="111"><div align="center"> <div align="center"><a href="../facilities.htm" class="titleLink">Facilities</strong></a></div></td>
                <td width="24"><div align="center"><font size="2">&nbsp;</font></div></td>
                <td width="118"><div align="center"><a href="../publications.htm" class="titleLink">Publications</a></div></td>
                <td width="22"><font size="2">&nbsp;</font></td>
                <td width="134"><div align="center"><a href="../contact.htm" class="titleLink">Contact Us</a></div></td>
              </tr>
            </table></td>
        </tr>
        <tr> 
          <td valign="top"><table width="100%" border="0" cellspacing="0" cellpadding="0">
              <tr> 
                <td width="16%" valign="top"><table width="919" height="275" border="0">
                  <tr>
                    <td width="79"></td>
                    <td width="495"><p><img src="images/MissWuxm.jpg" alt="My Picture" width="430" height="306"></p></td>
                    <td width="331">
						<p align="left" class="STYLE13">Wu, Xiaomei (<img src="images/wuxm_name.jpg" align=top>)</p>
						<p align="left" class="STYLE12"><strong>Ph.D candidate</strong></p>
						<p align="left" class="STYLE12"><strong> Expect to graduate in Jun. 2009</strong></p>
						<p align="left" class="STYLE12">&nbsp;</p>
						<p align="left" class="STYLE12"><span class="style17"><strong>Email:</strong> <a href="mailto:wuxm@mail.bnu.edu.cn">wuxm07@gmail.com</a><br />
	                        <strong>Phone:</strong> +86-10-5880 5045<br />
	                        <strong>Fax:</strong> +86-10-5880 7721<br />
                            <strong>Address:</strong> College  of Life Sciences, Beijing Normal  University, No.19, Xinjiekouwai Street, Beijing 100875, China. </span></p></td>
					</tr>
				</table>
				<ul class="STYLE1">
                    <table width="864" height="648" border="0">
                      <tr>
                        <td width="8">&nbsp;</td>
                        <td width="814" class="STYLE1" scope="row"><p align="left" class="STYLE9">EDUCATION</p>
                          <blockquote>
                            <p>Sep. 2004-Jul. 2009: Ph.D. in progress in Bioinformatics, 
    

     Laboratory of Computational Molecular Biology, College of Life Sciences,
    
Beijing Normal University.</p>
                            <p>Sep. 2000-Jul. 2004: B.S. in Computer Science and Technolog<span class="style14">y, <span class="style15">College of Information Science, </span>Beijing Normal University.</span></p>
                          </blockquote>                          
                          <p class="STYLE9"><strong>AWARDS &amp; SCHOLARSHIPS </strong></p>
                          <blockquote>
                            <p class="STYLE1">2005-2006 L iYun Scholarship for Excellent Graduates, Beijing Normal University (Second-class)</p>
                            <p>2006-2007 Academic Scholarship for Excellent Graduates, Beijing Normal University</p>
                          </blockquote>                          
                          <p><span class="STYLE9">RESEARCH EXPERIENCES </span></p>
                          <blockquote>
                            <p align="left"> Generally, I focus on applying the tools of bioinformatics, graph theory and statistics to the studies of biological networks and protein functions.</p>
                          </blockquote>
                          <ul>
                            <li>
                              <p align="left"><strong><span class="STYLE1">Developed</span> the project of 'genome-wide reconstruction of yeast protein-protein interaction network based on Gene Ontology' (Feb. 2005-Apr. 2006)</strong></p>
                            </li>
                            <p align="left">Solely derived from the Gene Ontology (GO), we have  defined an effective method, named Relative Specificity Similarity (RSS) for measuring the semantic similairty between two GO terms. Using the RSS method, we  reconstructed a yeast protein interaction network based on the GO and yeast annotations. Moreover, we obtained a gold standard positive (GSP) dataset and a gold standard negative (GSN) dataset of protein interactions. The two datasets, especially the GSN, are important for a statistical analysis system.</p>
                          </ul>                          <blockquote>
                            <p align="left" class="STYLE1"> Then, based on the release of GO (March 2006 ) and yeast annotations derived  from SGD (March 31, 2006), we developed a web-accessible database SPIDer (<i>Saccharomyces</i> protein-protein interaction database): <a href="http://cmb.bnu.edu.cn/SPIDer/index.html" target="_parent">http://cmb.bnu.edu.cn/SPIDer/index.html</a>.  SPIDer provides a variety of search options and graphical visualization of a network. Interestingly, users can retrieve the inter-member interactions in a list of proteins or in a protein complex of interest.</p>
                          </blockquote>
                          <ul>
                            <li><strong><span class="style15">Developing</span> the project of '<strong>characterizing hub proteins by projecting yeast protein-protein interaction network onto four cell cycle phases' </strong> <strong>(Nov. 2007-present, <em>ONGOING</em>)</strong></strong></li>
                          </ul>
                          <blockquote>
                            <p align="left" class="STYLE1">Currently, we focus on dynamic decomposition and analysis of yeast protein-protein interaction network. Through  temporally decomposing an integrated yeast protein interaction network into four cell cycle phases, we want to answer one question: 


 does a hub in the aggregated cell cycle interaction network still act as hubs in all or one of the four phase sub-networks?  Our main conclusion is that not all the hubs in the aggregated cell cycle network still act as hubs in all the decomposed phase sub-networks. The hubs observed in the aggregated network are just an assemblage of all the binary interactions of the hubs during different time series.</p>
                          </blockquote>                          <ul>
                            <li><b> <strong>Responsible for the project of </strong> <strong class="STYLE1" align="left">'</strong>GO-based functional categorization of genes<strong class="STYLE1" align="left">'</strong> <strong>(Jul. 2006-Jan. 2007)</strong></b></li>
                          </ul>
                          <blockquote>
                            <p align="left" class="STYLE1"> To obtain the best GO terms categorizing a list of genes of interest, several statistical methods are widely used, such as fisher's exact test and binomial test.  We tried to design an automatic approach using GO to help with the biological interpretation of the list of genes. Compared with the other tools using the classical methods, our approach unluckily shows no advantage.</p>
                          </blockquote>
                          <ul>
                            <li><strong class="STYLE1" align="left">Participated in the project of 'genome-wide inference of interacting motif pairs' (Oct. 2006-Dec. 2007)</strong></li>
                          </ul>
                          <blockquote>
                            <p align="left">An approach was designed to predicting high-confidence interacting motif pairs that are significantly over-represented in a high-confidence protein interaction (GSP) dataset compared to non-interaction (GSN) dataset.  The quality of four negative datasets was evaluated and compared. Of these, the one with the best performance was considered as the GSN dataset. In the project, one part of my work is to construct three negative datasets, two datasets derived from the RSS method and the other one comprising protein pairs in different subcellular compartments.</p>
                          </blockquote>                          
                          <ul>
                            <li><strong class="STYLE1" align="left">Participating in the project of '</strong><strong>classification of protein sequences by decomposing protein domain architecture network</strong><strong class="STYLE1" align="left">'</strong><strong> (May 2007-present, <em>ONGOING</em>)</strong></li>
                          </ul>
                          <blockquote>
                            <p align="left">Many current methods of classification of protein sequences are depend on sequence alignments, but establish incorrect families due to multiple domain features of proteins. To address this problem, we constructed a protein domain architecture network based on our previous work (Lin et al, Bioinformatics 2006), and designed a new method for protein classification by decomposing the network. The method was applied on 511,203 eukaryotic proteins which were dispatched into 6,797 protein groups. The groups were evaluated using the Eukaryotic Orthologous Groups of Proteins (KOGs) and GO annotations, indicating that proteins in each group evolutionarily manifest their homologous relationships. In the project, my work is mainly to evaluate the functional similarity within a protein group based on the GO annotations. </p>
                          </blockquote>                          <strong class="STYLE9" align="left">PUBLICATIONS</strong>                          <ol>
                              <li><strong>Wu, X.M.</strong>, Zhu, L, Guo, J, Zhang,D.Y and Lin, K.*.(2006) Prediction of yeast protein¨Cprotein interaction network: insights from the Gene Ontology and annotations. <strong><em>Nucleic Acids Research</em></strong>, 34(7):2137-2150. [<a href="http://nar.oxfordjournals.org/cgi/content/abstract/34/7/2137"
                              target="_blank">Online PDF</a>] </li>
                              <li><b>Wu, X.M.</b>, Zhu, L., Guo, J., Fu, C., Zhou, H.J, Dong, D., Li, Z.B, Zhang, D.Y. and Lin, K.* (2006) SPIDer: <i>Saccharomyces</i> protein-protein interaction database. <b><i>BMC Bioinformatics</i></b>, 7:S16. [<a
                               href="http://www.biomedcentral.com/1471-2105/7/S5/S16"
                               target="_blank">BioMed Central</a>]</li>
                              <li>Guo, J., <strong>Wu, X.M.</strong>, Zhang, D.Y. and Lin, K.* (2008) Genome-wide inference of protein interaction sites: lessons from the yeast high-quality negative protein¨Cprotein interaction dataset. <em><strong>Nucleic Acids Research</strong></em>, 36(6):2002-2011. [<a href="http://nar.oxfordjournals.org/cgi/content/short/36/6/2002" target="_blank">Online PDF</a>]</li>
                              <li><strong>Wu, X.M.</strong>, Guo, J., Zhang, D.Y. and Lin, K.* The properties of the hub proteins in yeast aggregated cell cycle network and its phase sub-networks (in submission). </li>
                              </ol>
                            <p class="STYLE9"><strong>ATTENDED COURSES &amp; CONFERENCES </strong></p>                          <ol>
                            <li>Nov. 5-7, 2004: '<strong>Machine Learning and Application</strong>' at Fudan  University, Shanghai, China. </li>
                            <li>Jun. 30-Jul. 4, 2005: Dragon-Star course titled '<strong>Computational Methods in Bioinformatics</strong>' at Jilin University, Changchun, China. The instructor: Dr. Dong Xu from University of Missouri.</li>
                            <li>Mar. 17-20, 2006: Short-time course titled '<strong>Bioinformatics Techniques, Methods and Applications</strong>'  sponsored by Beijing  Genomics and Bioinformatics   Center, held in Hangzhou, Zhejiang, China.</li>
                            <li>Aug. 24-27, 2006: '<strong>National Conference on Interdisciplinary Protein Research</strong>'   at Xiamen University, Fujian, China.</li>
                            <li>Dec. 18-20, 2006: '<strong>International Conference on Bioinformatics</strong>' <strong>(poster)</strong> organized by the Department of BioTechnology (DBT)

in New Delhi, Indian. </li>
                            <li> Mar. 11-13, 2007: '<strong>Molecular Phylogenetic Analysis and Application</strong>'  instructed by Yang Zhong from Fudan University, held at Beijing Normal University.</li>
                            <li> Apr. 9-19, 2007: '<strong>Spring Course on Population Genetics</strong>' instructed by Chung-I Wu from Chicago University of USA, held at Sun Yat-Sen University , Guangzhou, China.</li>
                            <li>May 12-16, 2008: '<strong>Doctor Forum of China in Bioinformatics</strong>'  <strong>(report)</strong> at Fudan University, Shanghai, China.</li>
                            <li>May 18-20, 2008: '<strong>CAS International Symposium on Developmental Systems Biology</strong>' organized by 


 Institute of Genetics and Developmental Biology, Chinese Academy of Sciences. </li>
                            <li>Oct. 6-9, 2008: '<strong>The Third National Conference on Bioinformatics and Systems Biology</strong>' <strong>(report)</strong> in Wuhan, China.</li>
                            </ol>
                          <p class="STYLE9"><strong>TECHING ASSISTANT</strong></p>
                          <ol>
                            <li>Sep. 2005-Jan. 2006: <strong>C/C++ programming language </strong> for undergraduate students, College of Life Sciences, Beijing Normal University.</li>
                            <li>Mar. 2007-Jul. 2007: <strong>Bioinformatics </strong> for undergraduate students, College of Life Sciences, Beijing Normal University.</li>
                          </ol>
                          <p class="STYLE9"><strong>TECHNICAL EXPERTISE </strong></p>
                          <ul>
                            <li>Operating systems: LINUX, WINDOWS</li>
                            <li>Programming languages: C/C++, PERL, HTML</li>
                            <li>Database system: MySQL</li>
                            <li>Statistical software: R, SPSS</li>
                          </ul>                          
                          <p>&nbsp;</p></td>
                        <td width="28">&nbsp;</td>
                      </tr>
                    </table>
                  </ul>
				</td>
              </tr>
            </table></td>
        </tr>
      </table></td>
  </tr>
  <tr> 
    <td height="1" bgcolor="#666666"></td>
  </tr>
  <tr> 
    <td height="60" bgcolor="#E4E4E4"><table width="100%" border="0" cellspacing="0" cellpadding="0">
        <tr> 
          <td width="16%">&nbsp;</td>
          <td width="635"><font size="-1">&copy; Laboratory of Computational Molecular Biology<br>
            College of Life Sciences, Beijing Normal University</font></td>
          <td width="153">&nbsp;</td>
        </tr>
      </table></td>
  </tr>
</table>

<!-- for google analytics, DO NOT delete -->
<script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
</script>
<script type="text/javascript">
_uacct = "UA-1045714-1";
urchinTracker();
</script>

</body>
</html>
